src/panels/config/zha/zha-groups-data-table.ts

```diff
@@ -0,0 +1,87 @@
+import "../../../components/data-table/ha-data-table";
+import "../../../components/entity/ha-state-icon";
+
+import memoizeOne from "memoize-one";
+
+import {
+  LitElement,
+  html,
+  TemplateResult,
+  property,
+  customElement,
+} from "lit-element";
+import { HomeAssistant } from "../../../types";
+// tslint:disable-next-line
+import { DataTableColumnContainer } from "../../../components/data-table/ha-data-table";
+// tslint:disable-next-line
+import { ZHAGroup, ZHADevice } from "../../../data/zha";
+import { formatAsPaddedHex } from "./functions";
+
+export interface GroupRowData extends ZHAGroup {
+  group?: GroupRowData;
+  id?: number;
+}
+
+@customElement("zha-groups-data-table")
+export class ZHAGroupsDataTable extends LitElement {
+  @property() public hass!: HomeAssistant;
+  @property() public narrow = false;
+  @property() public groups: ZHAGroup[] = [];
+  @property() public selectable = false;
```

Comment:
Is this also something that will be used in a future PR but is not used now?

Reply:
yes when group removal Is wired in

Reply:
Can we add it at that time?

Reply:
sure, I'm going to open it right after this is merged anyway

Reply:
done